nifi-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -58,14 +58,20 @@ dataflows.
 
 == Downloading and Installing NiFi
 
-NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads Page^]. There are two packaging options
-available: a "tarball" that is tailored more to Linux and a zip file that is more applicable for Windows users. Mac OS X users
-may also use the tarball or can install via Homebrew.
+WARNING: Before proceeding, check the Admin Guide to confirm you have the <<administration-guide.adoc#system_requirements,minimum system requirements>> to install and run NiFi.
 
-To install via Homebrew, simply run the command `brew install nifi`.
+NiFi can be downloaded from the link:http://nifi.apache.org/download.html[NiFi Downloads page^]. There are two packaging options
+available:
+
+- a "tarball" (tar.gz) that is tailored more to Linux
+- a zip file that is more applicable for Windows users
+
+Mac OS X users may also use the tarball or can install via link:https://brew.sh[Homebrew^] by simply running the command `brew install nifi` from the command line terminal.
+
+NOTE: NiFi requires Java 8 or 11. It is recommended that you have installed one of these Java versions prior to installing NiFi via Homebrew. Homebrew is aware of NiFi's Java dependency and may prompt to install an incompatible JDK as part of its NiFi install.
```

Comment:
Should this be "Homebrew is _unaware_ of NiFi's Java dependency and may prompt to install an incompatible JDK..." or "Homebrew is aware of NiFi's Java dependency and may prompt to install a _compatible_ JDK..."?

Reply:
Discussed with @andrewmlim , we agreed on this wording: "Homebrew is aware NiFi depends on Java but not the specific supported versions, so it may prompt to install an incompatible JDK as part of its NiFi install."